modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -347,10 +345,8 @@ DROP INDEX moviePlots
 [[vector-index-providers]]
 == Vector index providers for compatibility
 
-As of Neo4j 5.18, the default index provider is `vector-2.0`.
-As of Neo4j 5.26, it is no longer possible to specify an index provider when creating indexes.
-Instead, Neo4j selects the most performant provider (currently `vector-2.0`).
-
+It is not possible to specify an index provider when creating indexes in Cypher 25.
+Instead, Neo4j selects the most performant provider (currently `vector-2.0` for vector indexes).
 Previously created `vector-1.0` indexes will continue to function.
 
 .Learn more about vector index provider differences
```

Comment:
not really part of the removal of indexProvider but do we want to add a row in this table for the other config settings to say they are not available for `vector-1.0` but only for `vector-2.0`?
Something like:
```
| Indexed vector quantization and hnsw settings
| No support.
| Supported.
```
or if we want it more generic:
```
| Other indexed vector settings
| No support.
| Supported.
```
(or write out the settings fully, or split in one row per setting, ...)